javascript/pane.js

```diff
@@ -25,13 +25,17 @@ function Pane(options){
       context.show()
     })
 
-    // Show the pane if the user was tabbed onto the trigger and pressed enter or space
+    // Show the pane if the user was tabbed onto the trigger and pressed enter, space, or down arrow
     $(options.trigger).on('keyup', function(event){
       if ($(options.trigger).hasClass('disabled')) { return }
 
-      if (event.which == 13 || event.which == 32){
+      if (event.which === 32 || event.which === 40){
```

Comment:
Will we have an issue with the down arrow event being stopped (`return false`), when we're trying to use arrow keys to move down the list? How about the spacebar event being stopped while typing in the search input? I could imagine the spacebar event probably would be fine, but the down arrow may be an issue if there is no search field enabled. See 
https://github.com/culturecode/uber_select/blob/ebb8bd3f9a9350865221875cc78eca5ac12d3cdb/javascript/list.js#L10 and https://github.com/culturecode/uber_select/blob/ebb8bd3f9a9350865221875cc78eca5ac12d3cdb/javascript/uber_search.js#L55.

Reply:
When I was testing it didn't seem to be an issue, but perhaps you're aware of something I'm not on this one.